evm-contracts/scripts/native_solc6_compile

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This script generates .abi and .bin files for a selected .sol contract.
+# Example call:
+#   ./evm-contracts/scripts/native_solc_compile dev/Operator.sol
+#
+# The resulting abi and bin files are stored in ./evm-contracts/solc/v0.6
+
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; cd .. && pwd -P )"
+
+solc-select use 0.6.6
```

Comment:
What's the benefit of using solc-select over pulling down the solc binaries ourselves and referring to them explicitly?

Reply:
Ease of use, mainly. Saves us from having to commit a bunch of large files to the repo.

Reply:
I talked this over with @kaleofduty , and he points out that we avoid the potential supply-chain vulnerability if we pin the `pip install` to a hash, and audit its code at that commit. Can you do/check that, please? (I've looked over the source code, and it's very easy to audit.)

Reply:
@alexroan let's pin to a hash, it's good practice.

Reply:
Pinned it.